shared/pingpong/accounts.go

```diff
@@ -147,21 +150,30 @@ func prepareAssets(accounts map[string]uint64, client libgoal.Client, cfg PpConf
 			return
 		}
 
-		txid, broadcastErr := client.BroadcastTransaction(signedTxn)
-		if broadcastErr != nil {
-			fmt.Printf("Cannot broadcast asset creation txn\n")
-			err = broadcastErr
-			return
-		}
+		signedTrxs[i] = &signedTxn
 
 		if !cfg.Quiet {
-			fmt.Printf("Create a new asset: supply=%d, txid=%s\n", totalSupply, txid)
+			fmt.Printf("Create a new asset: supply=%d \n", totalSupply)
 		}
 		accounts[cfg.SrcAccount] -= tx.Fee.Raw
 	}
 
-	// get these assets
+	// submit the asset transactions and validate that they have all been accepted
 	for {
```

Comment:
It might be helpful to add a timeout for the outer for loop so that we don't stuck here forever.

Reply:
Removed ping-pong changes from this PR.